docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -351,19 +344,30 @@ def example_dlt_source(color: Optional[str] = None):
             ...
 
 
-@dlt_assets(
-    dlt_source=example_dlt_source(),
+dlt_source = example_dlt_source()
+dlt_pipeline = dlt.pipeline(
+    pipeline_name="example_pipeline_name",
+    dataset_name="example_dataset_name",
+    destination="snowflake",
+)
+
+
+class PartitionedRunDlt(RunDlt):
+    def stream(self, context: ComputationContext) -> Iterable:
+        color = context.partition_key
+        yield from DagsterDltResource().run(
+            context=context, dlt_source=example_dlt_source(color=color)
+        )
```

Comment:
This demonstrates what customization would look like. Subclassing instead of changing the body of the decorated function. This is actually marginally worse in this world, but the other advantages dominate.